### ISSUE TITLE
How to remove the line that runs down the editor

### ISSUE THREAD

**Issue by L0garithmic** [RechInformatica/rech-editor-batch#15]
https://i.imgur.com/rccXkC7.jpg

How do I turn this off?  It comes on when I enable the plugin

**Comment by BouwenMA** [comment]
Appears the creator is defining "editor.rulers" inside this extension itself that overrides values specified for user preferences.

I found I was able to remove the rulers by editing the extension's "package.json" file under:
    ".\VSCode\Insiders\App\Data\extensions\rechinformatica.rech-editor-batch-0.0.19"

- First I made a backup copy
- Next I removed the defined entry below, and saved the file:

```
"editor.rulers": [
	120
],

```
- Finally I then disabled the extension, restarted VSCode and re-enabled the extension.

The extension does not seem to provide any mechanism to override this via the normal preferences.json, so this is the only fix so far I know of that removes the ruler line until (if) the author provides an option.


Note: I also found several other editor settings defined in the extension that likely override user prefs (listed below).  I am unsure as to whether any of these are actually required for the extension to work.

```
    "editor.tabSize": 3,
    "editor.detectIndentation": true,
    "editor.trimAutoWhitespace": true,
    "editor.useTabStops": false,
    "editor.renderIndentGuides": false

```
